rss-client/src/app/User/Admin/admin-page/admin-page.component.html

```diff
@@ -1,17 +1,17 @@
 <!-- admin check -->
 <div *ngIf="user.admin">
-  <!-- main container -->
-  <div [ngSwitch]="view">
-    <!-- the same view can be shown in more than one case -->
-    <ng-template [ngSwitchCase]="'quiz'">
-      <admin-quiz (navigateEvent)="navigateTo($event)"></admin-quiz>
-    </ng-template>
-    <ng-template [ngSwitchCase]="'inventory'">
-      <admin-inventory (navigateEvent)="navigateTo($event)"></admin-inventory>
-    </ng-template>
-    <!--default case when there are no matches -->
-    <ng-template [ngSwitchDefault]>
-      <!-- <div class="container">
+    <!-- main container -->
+    <div [ngSwitch]="view">
+        <!-- the same view can be shown in more than one case -->
+        <ng-template [ngSwitchCase]="'quiz'">
+            <admin-quiz (navigateEvent)="navigateTo($event)"></admin-quiz>
+        </ng-template>
+        <ng-template [ngSwitchCase]="'inventory'">
+            <admin-inventory (navigateEvent)="navigateTo($event)"></admin-inventory>
+        </ng-template>
+        <!--default case when there are no matches -->
+        <ng-template [ngSwitchDefault]>
+            <!-- <div class="container">
       <!- quizzes 
```

Comment:
Was anything changed here besides tabs? If not please revert.

Reply:
we added a button so the admin can add a discount to a user